tests/sitl/test_localposition.py

```diff
@@ -0,0 +1,22 @@
+from dronekit import connect
+from dronekit.tools import with_sitl
+import time
+from nose.tools import assert_not_equals
+
+@with_sitl
+def test_timeout(connpath):
+    v = connect(connpath, await_params=True)
+
+    v.armed = True
+    v.flush()
+
+    time.sleep(5)
```

Comment:
I have not tried to tune it.  But I know that the boot sequence changes a lot.  Sitl used to calibrate the compass on boot for like 3 seconds.  Now it's faster.  But now the EKF needs to align on boot (which takes a while).  I used 5 seconds because sitl might take longer tomorrow than it does today, and that will be an annoying thing to debug.